MainDemo.Wpf/Buttons.xaml.cs

```diff
@@ -12,53 +13,56 @@
 using System.Windows.Media.Imaging;
 using System.Windows.Navigation;
 using System.Windows.Shapes;
+using System.Xml;
+using CodeDisplayer;
 using MaterialDesignColors.WpfExample.Domain;
+using MaterialDesignDemo.Helper;
 
 namespace MaterialDesignColors.WpfExample
 {
-    /// <summary>
```

Comment:
This seems to be just white space changes, please revert, it's just noise.

Reply:
Ok